lxd/storage/drivers/clients/alletra_wsapi1.go

```diff
@@ -33,6 +33,7 @@ const (
 
 	apiVolumeSetMemberAdd    = 1
 	apiVolumeSetMemberRemove = 2
+	apiActionGrowVolume      = 3
```

Comment:
It occurred to me now that we plan to add more appliance clients to this `clients` package, so these constants that are appliance specific should be renamed to prefix with `hpe`.

Reply:
I guess this kind of change even better to do as a single "fixup" commit. As it is just a trivial pattern rename.

Reply:
yep sounds good

Reply:
Fixed.